drivers/dma/dma_stm32.c

```diff
@@ -794,7 +809,10 @@ static void dma_stm32_config_irq_1(const struct device *dev)
 #endif /* DT_INST_IRQ_HAS_IDX(1, 6) */
 #endif /* DT_INST_IRQ_HAS_IDX(1, 7) */
 #endif /* CONFIG_DMA_STM32_SHARED_IRQS */
-/* Either 5 or 6 or 7 or 8 channels for DMA across all stm32 series. */
+/*
+ * Either 5 or 6 or 7 or 8 channels for DMA across all stm32 series.
+ * Cortex-M0 Socs: if dma2 exits, the channel interrupts overlap with dma1
```

Comment:
Instead of "Cortex-M0", please use exact references.
We cannot be 100% sure the next C-M0 based SoC will follow the same rule.

Reply:
done